packages/docusaurus-theme-classic/src/theme/TOCCollapsible/index.tsx

```diff
@@ -21,7 +21,6 @@ export default function TOCCollapsible({toc, className}: TOCCollapsibleProps) {
   return (
     <div
       className={clsx(
-        'margin-vert--md',
```

Comment:
Intentionally removing this utility class because it uses `important` rule, and I just want to avoid using `important` rule to overriding values in our stylesheet if possible.